### DISCUSSION TITLE
Example to reproduce same behaviour as v1 currency-input

### DISCUSSION THREAD

**Discussion by karladler** [dm4t2/vue-currency-input#272] [Q&A]
Is there any example for an input component reproducing the same behaviour as we had with the component from v1? I used the example from the docs, but it seems to have quite a few issues, when using :value and event emitters instead of v-model. Worked with v1 before just quite fine.

Using Vue2 with composition API

**Comment by dm4t2** [comment]
What exactly are the issues?

`<CurrencyInput :value="value" @input="value = $event" .../>` should work the same way as `<CurrencyInput v-model="value" .../>`.
